app/subscriber/src/features/my-products/MyProducts.tsx

```diff
@@ -98,10 +98,10 @@ export const MyProducts: React.FC = () => {
                 })}
             </div>
           </Show>
-          <Row className="page-section-title">
-            <FaUserPlus className="page-section-icon" /> Available products
-          </Row>
-          <p>
+          <h2 className="product-section-title">
+            <FaUserPlus /> <span>Available products</span>
```

Comment:
I've been guilty of this too, but we should wrap text in a div or span and not just leave it "hanging". Poses risk for whitespace styling errors

Reply:
ex)
Before the code was `<FaUserPlus /> Available Products` and another area was `<Icon />Test`. Could not figure out why the first had a `margin-right`, but in the end it was whitespace in the code.